src/libs/actions/BankAccounts.ts

```diff
@@ -62,6 +62,12 @@ function clearPlaid(): Promise<void | void[]> {
     return Onyx.set(ONYXKEYS.PLAID_DATA, CONST.PLAID.DEFAULT_DATA);
 }
 
+function clearInternationalBankAccount() {
+    return clearPlaid()
```

Comment:
Why we clear Plaid? I don't think it's required, as we don't interact with Plaid in this workflow